test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pytorchddp_inductor.py

```diff
@@ -46,7 +46,7 @@ def can_run_pytorchddp(ecr_image):
 @pytest.mark.skip_py2_containers
 @pytest.mark.efa()
 @pytest.mark.skip_inductor_test
-@pytest.mark.xfail(reason="known issue: https://github.com/pytorch/pytorch/issues/99074")
+@pytest.mark.skip_pt20_cuda121_tests
```

Comment:
did you check whether the issue above is no longer an issue in other SM images? The github issue looks like it was closed due to inactivity

Reply:
Just I removed that line, to make sure not to run it and fail it and skip it directly. It is not related to github issue being fixed or not

Reply:
actually I can add the github issue as a comment in next commit